src/middlewares/Validate.js

```diff
@@ -208,5 +208,24 @@ class Validate {
       check('reason', 'Reason should be a minimum of 2 letters').isString().isLength({ min: 2 }),
     ];
   }
+
+  /**
+  * Validate input
+* Validate accomodation inputs
+* @static
+* @returns {object} errors
+*/
+  static checkAccommodationRules() {
+    return [
+      check('name', 'name should be a minimum of 2 letters').isString().isLength({ min: 2 }),
+      check('address', 'address should be a minimum of 2 letters').isString().isLength({ min: 2 }),
+      check('description', 'description should be a minimum of 2 letters').isString().isLength({ min: 10 }),
+      check('rooms', 'Rooms must be provided').isLength({ min: 1 }),
+      check('cityId', 'cityId should be an integer').isInt(),
+      check('imageUrls', 'imageUrls must be provided').isLength({ min: 1 }),
```

Comment:
Oh, I think this is a bug!
It's only validating one Image if others have the wrong format.
<img width="1125" alt="Screen Shot 2019-12-06 at 09 45 58" src="https://user-images.githubusercontent.com/35301129/70305668-a3cd3e80-180d-11ea-9910-233b6a0dab8e.png">